src/transformers/models/dpr/modeling_tf_dpr.py

```diff
@@ -486,22 +486,26 @@ def serving(self, inputs):
 
             (a) For sequence pairs (for a pair title+text for example):
 
-                ``tokens: [CLS] is this jack ##son ##ville ? [SEP] no it is not . [SEP]``
+            ::
```

Comment:
That docstring had been changed to work around the doc-styling failure. This makes it the same as the PyTorch one,